core/services/transmission/signature.go

```diff
@@ -0,0 +1,19 @@
+package transmission
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+func SignMessage(
+	ownerPrivateKey *ecdsa.PrivateKey,
+	message []byte,
+) ([]byte, error) {
```

Comment:
Will this be expanded upon in the future? As of now it basically just wraps `crypto.Sign` right?

Reply:
Yeah I've used it locally for various purposes extra , might get expanded.